src/lib/mina_lib/conf_dir.ml

```diff
@@ -170,15 +170,22 @@ let export_logs_to_tar ?basename ~conf_dir =
     Option.value_map hw_file_opt ~default:base_files ~f:(fun hw_file ->
         hw_file :: base_files )
   in
-  let%map _result =
+  let tmp_dir = "/tmp" ^/ "mina-logs_" ^ basename in
```

Comment:
Should we use `Filename.temp_dir_name`?

Reply:
That's a good idea. There might be two users on the same machine who export logs at the same time.